lib/bloc/user_profile/currency.dart

```diff
@@ -6,35 +6,47 @@ import 'package:intl/number_symbols_data.dart';
 enum CurrencyID { BTC, SAT }
 
 class Currency extends Object {
-  final String symbol;
+  final String tickerSymbol;
   static const Currency BTC = Currency._internal("BTC");
```

Comment:
symbol is renamed to tickerSymbol as per ![image](https://user-images.githubusercontent.com/4012752/76475196-2cf8a080-640f-11ea-9ace-065a04fd6e26.png)